group1/comm/static/comm_tool.js

```diff
@@ -562,20 +562,31 @@ $(document).ready(function(){
   mobile_nav.sidebar();
 
   $('form#file_upload').submit(function(event){
-    $.ajax({
-      url: 'http://' + server_host + ':3000/upload',
-      type: 'POST',
-      data: new FormData( this ),
-      processData: false,
-      contentType: false,
-      success: function(file_path){
-        var download_url = 'http://' + server_host + ':' + server_port + '/' + file_path;
-        var display_name = $('input#filename').val();
-        $('input#text').val('<a href="' + download_url + '">' + display_name + '</a>' );
-        display();
-        $('#inputmodal').modal('hide');
-      }
-    });
+    var fileName = $('input#filename').val();
+    console.log(fileName);
```

Comment:
This was probably used for testing.  Would logging the filename to the console here without any more information be helpful when this code is deployed?  If you think this information could be useful, it might make sense to add supporting information so anybody looking at it would have some context.  If this was only used for debugging, it probably makes sense to remove it.